### ISSUE TITLE
Distopia blog post

### ISSUE THREAD

**Issue by hmacdope** [MDAnalysis/MDAnalysis.github.io#265]
I am planning on making a blog post about distopia that we can possibly also make a JOSS paper. 

I have it about half written and will hopefully have it done around the same time as the distopia code will be ready to be merged.

**Comment by orbeckst** [comment]
ping @hmacdope ;-)

**Comment by orbeckst** [comment]
(ok, fair enough, you have PR #269 up... so it's really "ping @richardjgowers ")

**Comment by orbeckst** [comment]
FYI – distopia was mentioned in the [2.9.0 release blog post](https://www.mdanalysis.org/2025/03/11/release-2.9.0/#enhancements).

**Comment by hmacdope** [comment]
Yep lets close this, I was planning to make a post on my personal blog with timings etc.
